helm/templates/entity-service-config.yaml

```diff
@@ -8,13 +8,32 @@ data:
   application.conf: |-
     entity.service.config = {
       entity-service {
+        dataStoreType = {{ .Values.entityServiceConfig.dataStoreType }}
+        {{- if eq .Values.entityServiceConfig.dataStoreType "mongo" }}
         mongo {
           {{- if .Values.entityServiceConfig.data.mongoUrl }}
           url = {{ .Values.entityServiceConfig.data.mongoUrl | quote }}
           {{- else }}
           host = {{ .Values.entityServiceConfig.data.mongoHost }}
           {{- end }}
         }
+        {{ else if eq .Values.entityServiceConfig.dataStoreType "postgres" }}
```

Comment:
Can this be simplified to pick all key/value pairs under the specified dataStoreType, instead of specific hard coding of specific keys

Reply:
I will do that. I choose this to be more readable in code vs documentation as few things are either or - e.g `url` or `host, port`.

Reply:
Done